querydsl-spatial/src/main/java/com/querydsl/spatial/jts/JTSPolygonExpression.java

```diff
@@ -40,7 +40,7 @@
     private volatile NumberExpression<Integer> numInteriorRing;
 
     @Nullable
-    private volatile JTSLineStringExpression<LineString> exterorRing;
+    private volatile JTSLineStringExpression<LineString> exteriorRing;
```

Comment:
**This is not backportable.**

We rely on default serialization, renames will break the serialized form.
I suggest either while cherry-picking removing this(be sure to mention that this change was not included), or splitting the commits.